packages/contracts/contracts/ChugSplashManager.sol

```diff
@@ -1023,7 +1027,11 @@ contract ChugSplashManager is OwnableUpgradeable, ReentrancyGuardUpgradeable {
      */
     function _deployContract(string memory _referenceName, bytes memory _code) internal {
         // Get the expected address of the contract.
-        address expectedAddress = Create2.compute(address(this), bytes32(0), _code);
+        address expectedAddress = Create2.computeAddress(
+            bytes32(0),
+            keccak256(_code),
+            address(this)
```

Comment:
nit: We can use [this version](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/Create2.sol#L44-L46) of `computeAddress`, which doesn't require a third argument